cmd/lakefs/cmd/run.go

```diff
@@ -398,6 +401,55 @@ func checkRepos(ctx context.Context, logger logging.Logger, authMetadataManager
 	}
 }
 
+func deleteExpiredAddressesJob(ctx context.Context, c *catalog.Catalog) {
+	s := gocron.NewScheduler(time.UTC)
+	_, err := s.Every(1).Hour().Do(func() {
```

Comment:
I think the duration should be tightly coupled with the token expiration time parameter (for example: 2 times the expiration time)

Reply:
done